packages/salesforce-adapter/src/filters/deploy_flows.ts

```diff
@@ -0,0 +1,75 @@
+/*
+*                      Copyright 2022 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import {
+  Change, ElemID, getChangeData, InstanceElement, isAdditionChange,
+  isInstanceChange,
+  isModificationChange, toChange,
+} from '@salto-io/adapter-api'
+import _, { isUndefined } from 'lodash'
+import { collections } from '@salto-io/lowerdash'
+import { RemoteFilterCreator } from '../filter'
+import { isInstanceOfType } from './utils'
+import { FLOW_METADATA_TYPE, SALESFORCE } from '../constants'
+import { isActivatingChange, isActivatingChangeOnly, isActiveFlowChange } from '../change_validators/flows'
+
+const { awu } = collections.asynciterable
+const ACTIVE = 'Active'
+
+const isRelevantChange = (change: Change<InstanceElement>): boolean => {
```

Comment:
I think this give more context than "relevant"
```suggestion
const isActivatingChange = (change: Change<InstanceElement>): boolean => {
```

Reply:
it is incorrect though, because it is not only activating change